pysb/tests/test_bng.py

```diff
@@ -23,6 +23,19 @@ def test_generate_network():
     ok_(generate_network(model))
 
 
+@with_model
+def test_generate_network_max_agg():
+    Monomer('A', ['b1', 'b1'])
+    Parameter('k', 1)
+    Parameter('A_0', 1)
+    Initial(A(b1=MultiState(None, None)), A_0)
+    Rule('a_bind_a', A(b1=MultiState(None, WILD)) + A(b1=MultiState(None, WILD)) >>
+         A(b1=MultiState(1, WILD)) % A(b1=MultiState(1, WILD)), k)
+    generate_equations(model, **{"max_stoich": {"A": 2}})
```

Comment:
Why not simply `generate_equations(model, max_stoich={"A": 2})`?

Reply:
that's right